### PR TITLE
change(release): Reduce the end of support time from 20 weeks to 16 weeks

### LINKED ISSUE #8521: Revert EOS halt period to 16 weeks
The EOS halt period was recently changed to 20 weeks.

We should revert this back to 16 weeks to match zcashd

### PR DESCRIPTION
**Pull request by arya2**
## Motivation

We want Zebra to have the same end of support timeline as zcashd.

Closes #8521.

#### Check before marking the PR as ready for review:
  - [x] Will the PR name make sense to users?
  - [x] Does the PR have a priority label?
  - [x] Have you added or updated tests?
  - [x] Is the documentation up to date?

##### For significant changes:
  - [x] Is there a summary in the CHANGELOG?
  - [x] Can these changes be split into multiple PRs?

_If a checkbox isn't relevant to the PR, mark it as done._

## Solution

Reduces `EOS_PANIC_AFTER` from 140 to 112 

## Review

Anyone can review.

### Reviewer Checklist

Check before approving the PR:
  - [ ] Does the PR scope match the ticket?
  - [ ] Are there enough tests to make sure it works? Do the tests cover the PR motivation?
  - [ ] Are all the PR blockers dealt with?
        PR blockers can be dealt with in new tickets or PRs.

_And check the PR Author checklist is complete._

### CODE REVIEW

**Review by oxarbitrage — COMMENTED** [review]
https://zebra.zfnd.org/dev/release-process.html#supported-releases

**Comment by oxarbitrage** [comment]
should we put this change in the top of the changelog for the next release now ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ All notable changes to Zebra are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org).
 
+## Zebra 1.X.X - XXXX-XX-XX
+
+### Changed
+
+- Adjust end of support from 20 to 16 weeks ([#8530](https://github.com/ZcashFoundation/zebra/pull/8530))
```
Comment (oxarbitrage) [inline CHANGELOG.md]:
```suggestion
- We realized that a longer than `zcashd` end of support could be problematic in some cases so we reverted back from 20 to 16 weeks ([#8530](https://github.com/ZcashFoundation/zebra/pull/8530))
```

**Review by oxarbitrage — DISMISSED** [review]
Thanks, i think it will be good to explain a bit more the change but i am ok as it is.

**Review by oxarbitrage — APPROVED** [review]
thanks!

**Comment by arya2** [comment]
@Mergifyio refresh

**Pull request merged**